src/groupByKey.ts

```diff
@@ -2,6 +2,16 @@ interface Item {
   [key: string]: any;
 }
 
-export function groupByKey(items, field) {
-  // write code here;
+export function groupByKey(items: Array<Item>, field: string): Item {
```

Comment:
```suggestion
export function groupByKey(items: Array<Item>, field: keyof Item): ItemGroup {
```
Використовуй keyof. В умові про це була підказка
Також зверни увагу, який тип повертає функція. Тобі потрібно повертати об'єкт типу
```
[key:string]: Item[]
```
Давай створимо окремий тип для цього